index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">  
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">  
+    <title>Survey Form</title>
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
```

Comment:
It would be nice if you put an h1 tag inside a header tag, it's considered the best practices 🤓

Reply:
- [ ] Well done! You are using professional communication and providing the why of your suggested changes. However in this case this should be a requested change instead of a suggestion because using [semantic HTML](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) is a requirement for HTML projects and as you have said it is a best practice.
- [ ]  Notice that if you want to mention something which is not required you can use the [OPTIONAL] notation

Reply:
Hello Eric, thanks for your suggestions, yes of course, but, I can't request a change because I did both the repo and the PR, so, GitHub doesn't let me mark as a _**" requested a change"**_